### PR TITLE
Add dockerfile and instructions for easier local dev

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

**Comment by sersut** [comment]
Didn't try them myself but sounds about right. 👍

**Comment by randomcamel** [comment]
Not related to your change, but it'd be cool to update the build-status badge now that the repo has moved from `sersut/` to `chef/`.

File: README.md [hunk 1]
```diff
@@ -32,41 +32,96 @@ Configure a webhook for your Github repository following the instructions [here]
...
-6. Install git: `apt-get install git`
-7. Put `lita-versioner` into `lita-dev/workspace`.
-8. Get your Jenkins API token by logging in to https://manhattan.ci.chef.co, clicking your name on the top right, and clicking Configure. Then click "Show API Token" and copy the values for the next step.
-9. Create `lita-versioner/lita_config.rb` with this text:
+### First Time Setup
+
+1. Install [Docker Toolbox](https://www.docker.com/products/docker-toolbox):
+   - `brew cask install dockertoolbox`
+   - `docker-machine create default --driver virtualbox`
+   - `eval $(docker-machine env default)` <-- you might want that in your .profile
```
Comment (randomcamel) [inline README.md]: `docker-machine start default`?

File: README.md [hunk 2]
```diff
@@ -32,41 +32,96 @@ Configure a webhook for your Github repository following the instructions [here]
...
+2. Copy `lita_config.rb.example` to `lita_config.rb` with this text:
    ```ruby
```
Comment (randomcamel) [inline README.md]: Broken formatting, the '```' has to be at the start of the line.

File: README.md [hunk 3]
```diff
@@ -32,41 +32,96 @@ Configure a webhook for your Github repository following the instructions [here]
...
+1. Set up the bot on Slack.
+   - Go to the [new Lita bot page](https://chefio.slack.com/apps/new/A0F7XDUJH-lita) on Slack.
+   - Create a username, like `<YOUR USERNAME>-testbot`.
+   - Copy the Slack token.
+2. Set up a channel for testing and invite your bot to it.
+   - /join #<YOUR USERNAME>-testbot
+   - /invite @<YOUR USERNAME>-testbot
+3. Configure the bot locally.
+   - Bring over the lita_config.rb.example slack configuration, inserting the token from step 1 and choosing a channel:
+     ```
```
Comment (randomcamel) [inline README.md]: '```' to start of line.

File: README.md [hunk 4]
```diff
@@ -32,41 +32,96 @@ Configure a webhook for your Github repository following the instructions [here]
...
+     config.default_inform_channel = "#<MY TEST CHANNEL>"
+     ```
+
+Now when you start the bot, it should join!
+
+### Testing Against Jenkins
+
+To test against Jenkins, you just need to add the correct tokens to the config file.
+
+1. Get your Jenkins API token by logging in to https://manhattan.ci.chef.co, clicking your name on the top right, and clicking Configure. Then click "Show API Token" and copy the values for the next step.
```
Comment (randomcamel) [inline README.md]: I think this is meant to be Step #4?

File: README.md (hunk 1 above)
Comment (jkeiser) [inline README.md]: Nope, `create` starts it.

File: README.md (hunk 2 above)
Comment (jkeiser) [inline README.md]: It actually works correctly indented, but you have to add a newline between. Fixed everywhere.

File: README.md (hunk 4 above)
Comment (jkeiser) [inline README.md]: I rearranged it so jenkins isn't part of the initial setup. No more reference to step 4 :)

File: README.md [hunk 5]
```diff
@@ -32,41 +32,91 @@ Configure a webhook for your Github repository following the instructions [here]
...
+
    ```ruby
    Lita.configure do |config|
-     config.handlers.versioner.jenkins_username = "YOUR USERNAME"
-     config.handlers.versioner.jenkins_api_token = "YOUR TOKEN"
-     config.robot.adapter = :shell
-     config.robot.log_level = :debug
+     # Add these two lines:
+     config.handlers.versioner.jenkins_username = "YOUR JENKINS USERNAME"
+     config.handlers.versioner.jenkins_api_token = "YOUR TOKEN FROM STEP 4"
```
Comment (randomcamel) [inline README.md]: Step 4.

File: README.md (hunk 1 above)
Comment (randomcamel) [inline README.md]: Must have been because I'd already done `create` and then shut the VM down once upon a time.

**Comment by randomcamel** [comment]
👍 I ran the instructions as far as getting a `lita` prompt.

**Pull request merged**